tests/data/dataset_readers/SQUAD_SAMPLE_FILE.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
yeah, json doesn't even like linebreaks not within a field...I think leaving it as a separate file is better for now